### PR TITLE
Advice about publication format regeneration

### PR DESCRIPTION
**Pull request by martinthomson**
The discussion at the interim highlighted that there were things we wanted to say about regeneration, but that we really didn't want to be prescriptive about it.  This attempts to balance the desire to stay out of it with the desire to not stay out of it.

### CODE REVIEW

File: draft-thomson-rswg-syntax-change.md [hunk 1]
```diff
@@ -133,9 +133,45 @@ This document has no IANA actions.
 
 --- back
 
+# Advice on Regenerating Publication Formats
+
+This document does not include specific guidance regarding the generation of
+publication formats from RFC XML source.  Decisions about how to maintain
+publication formats is not a matter governed by the policy in RFC 9280
```
Comment (ekr) [inline draft-thomson-rswg-syntax-change.md]:
```suggestion
publication formats are not a matter governed by policy as specified in RFC 9280
```

File: draft-thomson-rswg-syntax-change.md [hunk 2]
```diff
@@ -133,9 +133,45 @@ This document has no IANA actions.
...
+{{?RFC9280}}.  This section contains advice and considerations for the process
+of regeneration that came out of discussions of the policy changes in this
+document.
+
+Changes to the RFC XML for existing documents might result in changes to
+publication formats.  At the same time, the tools used to generate publication
```
Comment (ekr) [inline draft-thomson-rswg-syntax-change.md]: My argument here is that "HTML" is a publication format. That's not really metonymy for the published version

Comment (martinthomson) [reply draft-thomson-rswg-syntax-change.md]: I tried to capture that (rather subtle) distinction here, but probably failed.  Your point about generating, but not necessarily using, the HTML and whatnot was well made, I thought.

**Comment by ekr** [comment]
I think I would suggest using the words "rendered versions" rather than "publication formats" here.

**Pull request merged**